Android/ThirdPartyAuth/app/src/main/java/com/bentley/sample/thirdpartyauth/LoginViewModel.kt

```diff
@@ -12,44 +12,44 @@ import com.auth0.android.result.Credentials
 import com.bentley.sample.shared.MainActivity
 
 class LoginViewModel(
-	resourceHelper: IResourceHelper,
-	onAuth0TokenUpdated: (auth0Token: String)->Unit = {},
+        resourceHelper: IResourceHelper,
+        onAuth0TokenUpdated: (auth0Token: String) -> Unit = {},
```

Comment:
I don't think this should be double indented.

Reply:
Double-indentation makes sense to me, but it's coming from kotlin best practices. I can probably change it in the project style guide

Reply:
I took this out of context. I don't like the double-indent either. Let me see what's going on...